tools/write_inputs.sh

```diff
@@ -65,58 +68,96 @@ pushd $1
 
 popd
 
-####### modified function for sed -i
-function sedi { if [[ "$OSTYPE" == "darwin"* ]]; then
-        		sed -i '' "$1" "$2"
-		elif [[ "$OSTYPE" == "linux-gnu" ]]; then
-        		sed -i "$1" "$2"
-		fi;}
+if [ $start == "c" ]; then
 
+###### RUN MATLAB SCRIPT through HPC job script
+cat <<EOF > pre-job.$iexpnr
 
-####### set iexpnir in matlab file
-sedi "/expnr = '/s/.*/expnr = '$iexpnr';/g" $DA_TOOLSDIR"/write_inputs.m"
+#PBS -l walltime=24:00:00
+#PBS -l select=1:ncpus=8:mem=50gb
 
-###### RUN MATLAB SCRIPT
-cd $DA_TOOLSDIR/
-matlab -nodesktop -nosplash -r "write_inputs; quit"
-cd $DA_EXPDIR/$iexpnr
+module load tools/prod
+module load MATLAB/2023a_Update_3
+module load gcc/11.2.0
 
+cd $DA_TOOLSDIR
 
-###### alter files in namoptions
+export DA_TOOLSDIR=$DA_TOOLSDIR
+export DA_EXPDIR=$DA_EXPDIR
+export MATLAB_USE_USERWORK=1
```

Comment:
I thought we needed this to be 0, otherwise MATLAB starts in a `userpath` directory

```suggestion
export MATLAB_USE_USERWORK=0
```